### PR TITLE
updated mapping example to fix typos

### PR DESCRIPTION
**Pull request by terrycarlin**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @terrycarlin,

It seems you haven't yet signed a CLA. Please do so [here](https://cla.home-assistant.io/sign/terrycarlin?pr=1910).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_components/binary_sensor.iss.markdown [hunk 1]
```diff
@@ -37,6 +37,6 @@ The [generic camera platform](/components/camera.mjpeg/) offers the possibility
 camera:
   - platform: generic
     name: ISS
-    still_image_url: {% raw %}https://maps.googleapis.com/maps/api/staticmap?center={{ states.binary_sensor.iss.attributes.lat }},{{ states.binary_sensor.iss.attributes.long }}&zoom=5&size=500x500&maptype=roadmap&markers=color:blue%7Clabel:P%7C{{ states.binary_sensor.iss.attributes.lat }},{{ states.binary_sensor.iss.attributes.long }}{% endraw %}
+    still_image_url: {% raw %}https://maps.googleapis.com/maps/api/staticmap?center={{ states.binary_sensor.iss.attributes.latitude }},{{ states.binary_sensor.iss.attributes.longitude }}&zoom=5&size=500x500&maptype=roadmap&markers=color:blue%7Clabel:P%7C{{ states.binary_sensor.iss.attributes.latitude }},{{ states.binary_sensor.iss.attributes.longitude }}{% endraw %}
     limit_refetch_to_url_change: true
```
Comment (fabaff) [inline source/_components/binary_sensor.iss.markdown]: This change would only work if one is using `show_on_map: True` but not when running with the default. The idea was to enable the usage of ISS's location data but not show it on the map.

**Comment by terrycarlin** [comment]
is it possible to modify this component so that the attributes will be consistent or document that the attributes change their name depending on the value of “show_on_map”?

**Comment by robbiet480** [comment]
Hey, sorry the CLA link was wrong, it's now been fixed. Please sign the CLA [here](https://home-assistant.io/developers/cla_sign_start/).

**Comment by fabaff** [comment]
Updating the docs would be the simplest way. `ATTR_LONGITUDE` and `ATTR_LATITUDE` are picked up the frontend to show the position on the map automatically.

**Comment by andrey-git** [comment]
Any updates on this PR?

**Comment by fabaff** [comment]
This was addressed with https://github.com/home-assistant/home-assistant.github.io/commit/558bc3607aa5f3e72043ef8afc576c0778d6630c#diff-f090a3d1e0c1460c6693e818003e9cc9.

**Pull request closed without merging**